theano/sandbox/cuda/tests/test_fftconv.py

```diff
@@ -64,106 +64,6 @@ def test_full(self):
                       filters_shape=(2, 3, 3, 3),
                       border_mode='full', pad=True)
 
-    def test_opt_valid(self):
```

Comment:
Also, only the tests  theano.sandbox.cuda.tests.test_fftconv.TestConv2dFFT.test_opt_valid and  theano.sandbox.cuda.tests.test_fftconv.TestConv2dFFT.test_opt_full fail, but you remove 4 tests.